pyrometheus/chem_expr.py

```diff
@@ -153,6 +153,23 @@ def gen(c, t):
 
     return nasa7_conditional(p.Variable(arg_name), poly, gen)
 
+
+def constant_cp_enthalpy_expr(sp_thermo: ct.ConstantCp,
+                              arg_name) -> p.Expression:
+    t = p.Variable(arg_name)
+    c = sp_thermo.coeffs
+    c[1:] /= ct.gas_constant
```

Comment:
Will this expression _change_ the data in `sp_thermo.coeff`? 

```suggestion
c = sp_thermo.coeffs / ct.gas_constant
return c[1] / t + c[3] * (1 - c[0]*ct.gas_constant / t)
```

Reply:
@MTCam my bad, I lost sight of this. Good question--- I'm not sure, but your proposed change is cleaner.